taipy/core/config/scenario_config.py

```diff
@@ -159,9 +158,7 @@ def _to_dict(self) -> Dict[str, Any]:
         }
 
     @classmethod
-    def _from_dict(
-        cls, as_dict: Dict[str, Any], id: str, config: Optional[_Config] = None
-    ) -> "ScenarioConfig":  # type: ignore
+    def _from_dict(cls, as_dict: Dict[str, Any], id: str, config: Optional[_Config] = None) -> "ScenarioConfig":  # type: ignore
```

Comment:
I believe the comments should be counted in the line size.

Reply:
I will try to resolve this, probably in a different PR since it's a ruff thing.

Reply:
Not a major problem anyway.